src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import HomePage from "./HomePage";
+import homebtn from "../assets/home.png"
+
+export default function Header() {
+  return (
+    <header className="Header">
+      <NavLink to="/" element={<HomePage />}>
+      <img src={homebtn} alt="home"/>
+      </NavLink>
```

Comment:
In the app.js you define all the possible routes and which components you will be rendering in each case.

NavLink or Link is equivalent to <a> tag, so we only specify the url link here and Route does the job for us. So you do not need `element={<HomePage />}`